src/amp-story-player/amp-story-player-impl.js

```diff
@@ -1048,6 +1099,22 @@ export class AmpStoryPlayer {
     return this.iframes_[this.stories_[nextStoryIdx][IFRAME_IDX]];
   }
 
+  /**
+   * Checks if index is out of bounds
+   * @private
+   * @param {number} index
+   * @return {boolean}
+   */
+  isIndexOutofBounds_(index) {
```

Comment:
You could just `return this.currentIdx_ + index >= this.stories_.length || this.currentIdx_ + index < 0 ` instead of `if true return true else return false`